user_modules/IsReady_Template/ATF_Interface_IsReady_available_false_RAI_Template.lua

```diff
@@ -70,7 +70,7 @@ function StopStartSDL_HMI_MOBILE_VR_TTS(self)
 	--Start SDL
 	Test["Precondition_StartSDL_VR_TTS_available_false"] = function(self)
 
-		StartSDL(config.pathToSDL, config.ExitOnCrash)
+		StartSDL(commonPreconditions:GetPathToSDL(), config.ExitOnCrash)

```

Comment:
@istoimenova, GetPathToSDL() function is not required as a parameter for StartSDL() since both versions (with or without "/" at the end will work)

GetPathToSDL() function is useful in order to create concatenated path to particular file